### PR TITLE
WIP: telcov10n: add testing phase and tests script to hcp testing

### PR DESCRIPTION
**Pull request by rshemtov13**
This PR will add the testing portion of our (telcov10n) Hypershift testing for NTO test cases For now we will only be testing tier-0 test cases specified in the commands.sh script. After the results have been stable for multiple runs we will gradually progress to the entire test suite.

### CODE REVIEW

File: ci-operator/step-registry/telcov10n/functional/hcp-cnf/telcov10n-functional-hcp-cnf-workflow.yaml [hunk 1]
```diff
@@ -4,6 +4,8 @@ workflow:
     pre:
       - ref: telco5g-jobs-router
       - ref: telco5g-hcp-cluster-setup
+    test:
```
Comment (sshnaidm) [inline ci-operator/step-registry/telcov10n/functional/hcp-cnf/telcov10n-functional-hcp-cnf-workflow.yaml:7]: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/58977/pull-ci-openshift-release-master-ci-operator-config/1858858108537802752

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-hcp-cnf-4.17-e2e-telcov10n-functional-hcp-cnf-stable

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-hcp-cnf-4.17-e2e-telcov10n-functional-hcp-cnf-stable

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-hcp-cnf-4.17-e2e-telcov10n-functional-hcp-cnf-stable

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-hcp-cnf-4.17-e2e-telcov10n-functional-hcp-cnf-stable

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-hcp-cnf-4.17-e2e-telcov10n-functional-hcp-cnf-stable

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-hcp-cnf-4.17-e2e-telcov10n-functional-hcp-cnf-stable

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-hcp-cnf-4.17-e2e-telcov10n-functional-hcp-cnf-stable

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-hcp-cnf-4.17-e2e-telcov10n-functional-hcp-cnf-stable

**Comment by mrniranjan** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-hcp-cnf-4.17-e2e-telcov10n-functional-hcp-cnf-stable

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-hcp-cnf-4.17-e2e-telcov10n-functional-hcp-cnf-stable

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-hcp-cnf-4.17-e2e-telcov10n-functional-hcp-cnf-stable

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-hcp-cnf-4.17-e2e-telcov10n-functional-hcp-cnf-stable

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-hcp-cnf-4.17-e2e-telcov10n-functional-hcp-cnf-stable

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-hcp-cnf-4.17-e2e-telcov10n-functional-hcp-cnf-stable

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-hcp-cnf-4.17-e2e-telcov10n-functional-hcp-cnf-stable

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-hcp-cnf-4.17-e2e-telcov10n-functional-hcp-cnf-stable

**Pull request closed without merging**